lib/Config.js

```diff
@@ -150,16 +150,21 @@ class Config {
                 cert: certpath,
             };
         } else if (key || cert) {
-            throw new Error( 'bad config: both certFilePaths.key and ' +
+            throw new Error('bad config: both certFilePaths.key and ' +
                 'certFilePaths.cert must be defined');
         }
         /**
          * Configure the backends for Authentication, Data and Metadata.
          */
-        let auth = '';
-        let data = '';
-        let metadata = '';
+        let auth = 'mem';
+        let data = 'file';
+        let metadata = 'file';
         if (process.env.S3BACKEND) {
+            const validBackends = ['mem', 'file', 'scality'];
+            assert(validBackends.indexOf(process.env.S3BACKEND) > -1,
+                'bad environment variable: S3BACKEND environment variable ' +
+                'should be one of mem/file/scality'
+            );
             auth = process.env.S3BACKEND;
             data = process.env.S3BACKEND;
             metadata = process.env.S3BACKEND;
```

Comment:
This will set everything to the same backend. Say that i want file auth and scality metadat, this won't allow it. Is taht intended?